src/LoginForm/LoginForm.js

```diff
@@ -30,24 +31,30 @@ export class LoginForm extends Component {
 
   verifyCredentials = (e) => {
     e.preventDefault();
-    if (!this.state.userName && !this.state.password) {
+    if (!this.state.userName && !this.state.password && !this.state.email) {
       return false
     } else {
 
       const credentials = {
         email : this.state.email,
         password : this.state.password
       }
+      
       this.props.authenticateUser(credentials);
       this.clearInputs();
+ 
     }
   }
 
   render(){
     const {userName, email, password} = this.state;
+
     return (
 
       <form className="form-container">
+        <Link to='/'>
```

Comment:
I know this is in reference to the homepage. Should it have a descriptive name? Or does this pass accessibility? We can ask a mentor/instructor when we get the chance.